qiime/plugin/testing.py

```diff
@@ -0,0 +1,87 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016--, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import pkg_resources
+import tempfile
+import unittest
+import qiime
+
+
+# TODO Split out into more specific subclasses if necessary.
+class TestPluginBase(unittest.TestCase):
+    package = None
+
+    def setUp(self):
+        plugin = None
+        package = None
```

Comment:
I'm not sure if this is necessary --- we don't need `package` if the `try/except` starting on 24 fails.